rfcs/README.md

```diff
@@ -2,14 +2,58 @@
 
 We're trying a new approach of checking in Fluent UI RFCs to the repo so that they're more persistent, easier to find, and we can use PR comments to discuss the proposals.
 
+## Why RFCs?
+
+The RFC process is intended to bring a few benefits:
+
+- More efficent and durable decision making
+  - **Efficent** as we can all engage and review on our own time, rather than locking X engineers in a room for hours
+  - **Durable** as we give more time for decisions to be reviewed, commented, and considered. Rather than making a decision
+    after being locked in a room for an hour with X other engineers.
```

Comment:
nit: These two should go in the same line.

Reply:
fixed